### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by soshimee** [Lauriethefish/BetterPortals#151]
**Describe the bug**
Breaks randomly

**To Reproduce**
Make a portal

**Server Logs**
```
- [Thu 04:43:39 ERROR BetterPortals] A critical error occurred during plugin execution.
- [Thu 04:43:39 ERROR BetterPortals] Please create an issue at https://github.com/Lauriethefish/BetterPortals/issues to get this fixed.
! [Thu 04:43:39 WARN ] FieldAccessException: Field index out of bounds. (Index: 5, Size: 5)
! [Thu 04:43:39 WARN ]     at ProtocolLib (1).jar//com.comphenix.protocol.reflect.StructureModifier.writeInternal(StructureModifier.java:371)
! [Thu 04:43:39 WARN ]     at ProtocolLib (1).jar//com.comphenix.protocol.reflect.StructureModifier.write(StructureModifier.java:347)
! [Thu 04:43:39 WARN ]     at BetterPortals-0.11.1.jar//com.lauriethefish.betterportals.bukkit.entity.faking.EntityPacketManipulator.setSpawnRotation(EntityPacketManipulator.java:137)
! [Thu 04:43:39 WARN ]     at BetterPortals-0.11.1.jar//com.lauriethefish.betterportals.bukkit.entity.faking.EntityPacketManipulator.showEntity(EntityPacketManipulator.java:58)
! [Thu 04:43:39 WARN ]     at BetterPortals-0.11.1.jar//com.lauriethefish.betterportals.bukkit.entity.faking.IEntityPacketManipulator.showEntity(IEntityPacketManipulator.java:28)
! [Thu 04:43:39 WARN ]     at BetterPortals-0.11.1.jar//com.lauriethefish.betterportals.bukkit.entity.faking.EntityTracker.addTracking(EntityTracker.java:157)
! [Thu 04:43:39 WARN ]     at BetterPortals-0.11.1.jar//com.lauriethefish.betterportals.bukkit.entity.faking.EntityTrackingManager.setTracking(EntityTrackingManager.java:43)
! [Thu 04:43:39 WARN ]     at BetterPortals-0.11.1.jar//com.lauriethefish.betterportals.bukkit.player.view.entity.PlayerEntityView.updateReplicatedEntities(PlayerEntityView.java:89)
! [Thu 04:43:39 WARN ]     at BetterPortals-0.11.1.jar//com.lauriethefish.betterportals.bukkit.player.view.entity.PlayerEntityView.update(PlayerEntityView.java:42)
! [Thu 04:43:39 WARN ]     at BetterPortals-0.11.1.jar//com.lauriethefish.betterportals.bukkit.player.view.PlayerPortalView.update(PlayerPortalView.java:68)
! [Thu 04:43:39 WARN ]     at BetterPortals-0.11.1.jar//com.lauriethefish.betterportals.bukkit.player.PlayerData.updatePortalViews(PlayerData.java:79)
! [Thu 04:43:39 WARN ]     at BetterPortals-0.11.1.jar//com.lauriethefish.betterportals.bukkit.player.PlayerData.onUpdate(PlayerData.java:111)
! [Thu 04:43:39 WARN ]     at java.base/java.util.HashMap$Values.forEach(HashMap.java:1067)
! [Thu 04:43:39 WARN ]     at java.base/java.util.Collections$UnmodifiableCollection.forEach(Collections.java:1092)
! [Thu 04:43:39 WARN ]     at BetterPortals-0.11.1.jar//com.lauriethefish.betterportals.bukkit.tasks.MainUpdate.run(MainUpdate.java:47)
! [Thu 04:43:39 WARN ]     at org.bukkit.craftbukkit.v1_19_R1.scheduler.CraftTask.run(CraftTask.java:101)
! [Thu 04:43:39 WARN ]     at org.bukkit.craftbukkit.v1_19_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:483)
! [Thu 04:43:39 WARN ]     at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1490)
! [Thu 04:43:39 WARN ]     at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:446)
! [Thu 04:43:39 WARN ]     at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1414)
! [Thu 04:43:39 WARN ]     at net.minecraft.server.MinecraftServer.v(MinecraftServer.java:1187)
! [Thu 04:43:39 WARN ]     at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:302)
! [Thu 04:43:39 WARN ]     at java.base/java.lang.Thread.run(Thread.java:831)
  [Thu 04:43:39 INFO  BetterPortals] Attempting to shut down the plugin . . .
  [Thu 04:43:39 INFO  BetterPortals] Disabling BetterPortals v0.11.1
```

**Comment by soshimee** [comment]
Can there not be an option to ignore errors (not shut down even if error)? The plugin works, until this appears.

**Comment by HarvelsX** [comment]
@soshimee, try disabling entity support: `enableEntitySupport` to `false`

**Comment by Lauriethefish** [comment]
Fixed in 3b4a377faa962267149ce472a4f48cf128f5cebb
